internal/ingress/controller/controller.go

```diff
@@ -1753,6 +1750,7 @@ func externalNamePorts(name string, svc *apiv1.Service) *apiv1.ServicePort {
 	}
 
 	for _, svcPort := range svc.Spec.Ports {
+		//nolint:gosec // Ingore G109 error
```

Comment:
Typo on ignore / ingore

Also, what is error G109? Can we explain why this is being ignored? :)

Reply:
The misspell linter seems not working for the nolint comment :P
The error G109 is: `Potential Integer overflow made by strconv.Atoi result conversion to int16/32 (gosec)`